docs/src/test/scala/docs/http/scaladsl/server/directives/JwtDirectivesExamplesSpec.scala

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (C) 2009-2023 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package docs.http.scaladsl.server.directives
+
+import akka.http.jwt.scaladsl.server.directives.JwtDirectives.jwt
+import akka.http.scaladsl.model.headers._
+import akka.http.scaladsl.server.{ AuthorizationFailedRejection, RoutingSpec }
+import docs.CompileOnlySpec
+
+class JwtDirectivesExamplesSpec extends RoutingSpec with CompileOnlySpec {
+  "jwt" in {
+    //#jwt
+    val route =
+      jwt() {
+        _.stringClaim("role") match {
+          case Some("admin") => complete(s"You're in!")
+          case None          => reject(AuthorizationFailedRejection)
+        }
+      }
+
+    // tests:
+
+    // regular request
+
+    // manually injected valid JWT for test purposes with a claim "role" -> "admin"
+    val jwtToken = Authorization(OAuth2BearerToken(
+      "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIn0.Gfx6VO9tcxwk6xqx9yYzSfebfeakZp5JYIgP_edcw_A"))
```

Comment:
Should we use an hardcoded token here? I started with building the token in-place but the solution to build one with signature end up a bit clunky (unless I exposed the utility we use ourselves internally which didn't seem like a good option)... ended up leaving it like this but still wondering..

Reply:
I think this is fine